src/types.ts

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { OutputOptions, TransformSourceDescription, PluginContext } from 'rollup';
+
+// @see https://github.com/estree/estree/blob/master/es2015.md#exports
+export const EXPORT_NAMED_DECLARATION = 'ExportNamedDeclaration';
+export const EXPORT_SPECIFIER = 'ExportSpecifier';
+export const EXPORT_DEFAULT_DECLARATION = 'ExportDefaultDeclaration';
+export const EXPORT_ALL_DECLARATION = 'ExportAllDeclaration';
+export const ALL_EXPORT_TYPES = [
+  EXPORT_NAMED_DECLARATION,
+  EXPORT_SPECIFIER,
+  EXPORT_DEFAULT_DECLARATION,
+  EXPORT_ALL_DECLARATION,
+];
+
+export enum ExportClosureMapping {
```

Comment:
can we test for `export class` and `export default class`

Reply:
`export class` support added in e05f26c.

Reply:
`export default class` support added in 9bd28f3.